src/app/components/NavigationBar/NavigationBar.blocks.tsx

```diff
@@ -277,11 +277,13 @@ export const NavigationBarFull: React.FC<NavigationBarFullProperties> = ({
 			return [];
 		}
 
+		const enabledNetworkIds = profileAllEnabledNetworkIds(profile);
+
 		return profile
 			.wallets()
 			.values()
-			.filter((wallet) => availableNetworks.some((networkItem) => wallet.network().id() === networkItem.id()));
-	}, [profile, isProfileRestored, availableNetworks]);
+			.filter((wallet) => enabledNetworkIds.includes(wallet.network().id()));
+	}, [profile, isProfileRestored, profileAllEnabledNetworkIds]);
```

Comment:
```suggestion
	}, [profile, isProfileRestored, profileAllEnabledNetworkIds, enabledNetworkIds]);

```

Reply:
Correction: `profileAllEnabledNetworkIds` is not required

```suggestion
	}, [profile, isProfileRestored, enabledNetworkIds]);
```